All EIPIP Meetings/Meeting 030.md

```diff
@@ -0,0 +1,85 @@
+### [Audio/Video of the meeting](https://youtu.be/FdCeXxok2Fw) 
+### Moderator: Pooja Ranjan
+### Notes: Pedro Rivera 
+### [Previous Call (29)](https://github.com/ethereum-cat-herders/EIPIP/blob/master/All%20EIPIP%20Meetings/Meeting%20029.md)
+
+## Decisions Made
+
+| Decision Item | Description | Video ref |
+| ------------- | ----------- | --------- |
+| **30.1**      | Split EIP and ERC repos and minimize number overlaps by offsetting ERC and EIP numbers by an amount in the thousands. | [5:40](https://youtu.be/FdCeXxok2Fw?t=340) |     
+| **30.2**      | JSON RPC spec shall be documented under eth1.0-specs/json-rpc | [35:10](https://youtu.be/FdCeXxok2Fw?t=2110) | 
+| **30.3**      | Remove "testing greelight" from EIP process. | [43:15](https://youtu.be/FdCeXxok2Fw?t=2595)  | 
+
+## Action Items
+
+| Action Item | Description |
+| ------------| ----------- |
+| **30.1**    | Alita to take on the solution for minimizing EIP/ERC number overlaps.|
+| **30.2**    | Spread the word and find an entry-level resource to contiribute to the JSON RPC spec.|
+| **30.3**    | Tim Beiko will bring up the agreed changes to the EIP process in the All Core Devs meeting.|
+-----------------------------
+
+## Splitting out EIP repos into EIPs & ERCs
+
+* **Pooja** introduces the topic, which stems from the decision made in the previous meeting. The agreement was to have EIP editors that are separate from ERC editors in order to split notifications to editors of EIP/ERC. However, one potential conflict would be numbering, as some people want to avoid overlaps.
+* **Micah** stated his weak preference for simply allowing collisions, but then recognized the downside is multiple cases of user confusion. 
+* **Hudson** voiced his preference to mitigate overlaps by offsetting the ERCs/EIPs by 1000 or so. **Micah** pointed out that to do this some solution to pick numbers must be devised. One option is to burn through issue numbers 1-N through a script to induce the offset. **Pooja** suggested an alternative, however, it does not allow for independent subscription to ERC and EIP. 
+* **Micah** mentions again it could be easy to write a script that burns through 1-3500 on ERCs repo, and that it is probably the easiest and satisfying the most people. Would only need to find someone to do it. Alita could pick it up, since she voiced her interest.
+* **lightclient** objects he still feels there should not be overlaps between ERCs and EIPs. Proposes some central source or having another repo to create an issue to create a number so they happen sequentially. **Micah** inquires about his concern and, after a brief discussion, it is concluded that the decisions we take here don't matter much long term, so, it's okay if we take on some short term technical debt because lightclient's concerns will likely disappear in the future (eventually the EIP process will change, but until that happens we still do want them split apart).
```

Comment:
```suggestion
* **lightclient** objects he still feels there should not be overlaps between ERCs and EIPs. Proposes some central source or having another repo to create an issue to create a number so they happen sequentially. 
**Micah** inquires about his concern and, after a brief discussion, it is concluded that the decisions we take here don't matter much long term, so, it's okay if we take on some short term technical debt because lightclient's concerns will likely disappear in the future (eventually the EIP process will change, but until that happens we still do want them split apart).
```